CGAL_ImageIO/include/CGAL/Image_3.h

```diff
@@ -150,6 +150,10 @@ class CGAL_IMAGEIO_EXPORT Image_3
   float ty() const { return image_ptr->ty; }
   float tz() const { return image_ptr->tz; }
 
+  float& tx(){ return image_ptr->tx; }
+  float& ty(){ return image_ptr->ty; }
+  float& tz(){ return image_ptr->tz; }
+
```

Comment:
That modification is not consistent with the rest of the file.

If that is not a hard requirement, it should be reverted.

Reply:
We need a way to override the origin in case it is not in the file.   Alternatively we might add parameters to the constructor.